validate.py

```diff
@@ -92,6 +96,10 @@ def _validate(
     info: Info,
     index_url: str,
 ) -> None:
+    # quick hack before i go to bed - py_exe just prepends python to {major}.{minor}
+    if python[6:] not in info.python_versions:
```

Comment:
assuming this is unacceptable i plan on making this nicer and breaking it out into a separate pr tomorrow